consumer/console.go

```diff
@@ -53,6 +55,11 @@ const (
 // pipe is closed, i.e. when EOF is detected.
 // By default this paramater is set to "true".
 //
+// - SetMetadata: If this value is set to "true" the `Metadata` will set to each message
```

Comment:
Better: "When this value is set to true, the fields mentioned in the metadata section will be added to each message. Adding metadata will have a performance impact on systems with high throughput."

Same for the other occurrences of this setting.